package.json

```diff
@@ -28,8 +28,10 @@
   "dependencies": {
```

Comment:
Could you also:
- change "format" to just cheking
- add "format:fix"
- add "lint": "npm run format && npm rum eslint && npm run stylelint"

This is all usually done for automation tools like "CI/CD" that should have "just check, no mutation" option

Reply:
Additionally, be sure to re-run "lint:fix" add add to the commit all the affected files if any.